crates/re_types_builder/src/lib.rs

```diff
@@ -427,6 +427,7 @@ pub(crate) fn to_snake_case(s: &str) -> String {
 
     let mut parts: Vec<_> = s.split('.').map(ToOwned::to_owned).collect();
     if let Some(last) = parts.last_mut() {
+        *last = last.replace("UVec", "uvec");
```

Comment:
😬 it would be nice if we picked an algorithm that did the right thing here instead (i.e. ignoring Upper->Upper transitions), but I guess we shouldn't waste too much time on that right now

Reply:
Yeah as long as we have the expected output I'm happy to be honest; this is just a build time thing.